src/App.tsx

```diff
@@ -14,11 +15,98 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  addedGoods: string[],
+};
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    addedGoods: ['Jam'],
+  };
+
+  render() {
+    const array = this.state.addedGoods.join(', ');
+
+    return (
+      <div className="App p-4 d-flex">
+        <div className="w-50">
+          <ul className="list-group mb-3">
+            {goodsFromServer.map(good => (
+              <li
+                key={good}
+                className={this.state.addedGoods.includes(good)
```

Comment:
Will be better to store the result of includes method in some variable
Now you call the same function at least 3 times in your code